cmake/Buildosqp-eigen.cmake

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) Fondazione Istituto Italiano di Tecnologia
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(osqp QUIET)
+
+ycm_ep_helper(osqp-matlab TYPE GIT
+              STYLE GITHUB
+              REPOSITORY dic-iit/osqp-matlab.git
+              TAG master
```

Comment:
ither change the branch there to `master`, or here to `main`. Out of curiosity, why `main`?

Reply:
I changed to main. main is the new default branch name in GitHub : https://github.com/github/renaming#new-repositories-use-main-as-the-default-branch-name .

Reply:
Thx! good to know.